spec/models/comment_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  subject { Comment.new(author_id: 3, post_id: 3) }
+
+  context 'it should exist' do
+    it 'should exist as instance' do
+      expect(subject).to be_instance_of Comment
+    end
+  end
```

Comment:
- [ ] kindly add test for `updatecommentscounter` method in your comment model

Reply:
@VICTOR-CHUKWU  done